### PR TITLE
Adds algorithm specification for KAS-ECC CDH-Component Sp800-56Ar3

### PR DESCRIPTION
**Pull request by Kritner**

### CODE REVIEW

File: src/kas/sp800-56ar2/ecc/sections/08-ecc-cdh-comp.adoc [hunk 1]
```diff
@@ -36,7 +36,7 @@ The following is a example JSON object advertising support for KAS ECC CDH-Compo
 		"valValue": "123456"
 	}],
 	"function": ["keyPairGen", "dpGen"],
-	"curve": ["p-192", "k-163", "b-163"]
+	"curve": ["p-224", "k-233", "b-233"]
```
Comment (bfussell) [inline src/kas/sp800-56ar2/ecc/sections/08-ecc-cdh-comp.adoc:39]: Are these examples accurate ? I think all other curve strings use upper case, eg P-224

File: src/kas/sp800-56ar3/ecc/sections/08-ecc-cdh-comp.adoc [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+----
+{
+	"algorithm": "KAS-ECC",
+	"mode": "CDH-Component",
+	"revision": "Sp800-56Ar3",
+	"prereqVals": [{
+		"algorithm": "ECDSA",
+		"valValue": "123456"
+	}],
+	"function": ["keyPairGen", "dpGen"],
```
Comment (bfussell) [inline src/kas/sp800-56ar3/ecc/sections/08-ecc-cdh-comp.adoc:38]: Is there a place where all possible values for "function" are defined ?

**Review by bfussell — COMMENTED** [review]
Sorry for the late review.

**Comment by Kritner** [comment]
Whoops, I didn't mean to resolve, I'll take care of those in another PR

**Comment by Kritner** [comment]
This change is on prod as of release [v1.1.0.17](https://github.com/usnistgov/ACVP-Server/releases/tag/v1.1.0.17)

**Pull request merged**